wfl/DBManager.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Collections.Generic;
+using System.Configuration;
+using System.Threading.Tasks;
+using MongoDB.Driver;
+using wfl.Models;
+
+namespace wfl
+{
+    public class DBManager
+    {
+        private static string connectionString;
+        public static MongoClient client;
+        public static IMongoDatabase db;
+
+        static DBManager()
+        {
+            connectionString = ConfigurationManager.ConnectionStrings["databaseConnectionString"].ConnectionString;
```

Comment:
While reading the .net docs I noticed WebConfigurationManager as the preferred way to work with configuration files related to Web applications.  Should that be used here instead of ConfigurationManager?